### DISCUSSION TITLE
Old versions of VAPIX API Library

### DISCUSSION THREAD

**Discussion by filipeandrade6** [AxisCommunications/discussions#26] [VAPIX] [answered]
Hello,

Where do I find the old versions of the VAPIX API Library?

I manage a lot of old Axis cameras models that doesn't have the new version of the VAPIX API Library. I'm writting some automation scripts that would be incredible useful to use the API.

Thanks!

**Comment by filipeandrade6** [comment]
sending a http request to the url `http://<camera-ip>/axis-cgi/param.cgi?action=list` I get list of the camera settings, it will work for me but if there is any documentation of the older versions of the VAPIX API would be awesome.

Thanks again!

**Comment by Corallo** [comment, marked as answer]
Hello @filipeandrade6 

The only page with vapix documentation that we have is  [this](https://www.axis.com/vapix-library/) 
Asking around, I found this older version, hope it helps
[VAPIX_2_Parameter_Specs.pdf](https://github.com/AxisCommunications/discussions/files/9242789/VAPIX_2_Parameter_Specs.pdf)

**Reply by filipeandrade6** [reply]
Hi @Corallo!

I'll check it out the PDF file.

Thanks for your reply, have a good day!
